alfresco-java-rest-api-common/src/main/java/org/alfresco/rest/sdk/feign/oauth2/OAuth2FeignRequestInterceptor.java

```diff
@@ -17,45 +17,189 @@
 
 import static org.springframework.http.HttpHeaders.AUTHORIZATION;
 
-import feign.RequestInterceptor;
-import feign.RequestTemplate;
+import java.time.Instant;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.security.oauth2.client.OAuth2AuthorizeRequest;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
 import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientService;
 import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.security.oauth2.core.OAuth2RefreshToken;
+
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
 
 /**
  * Feign {@link RequestInterceptor} that makes use of the OAuth2 support classes from Spring Security to obtain an access token and add the corresponding
  * authorization header to the feign request.
  */
-public class OAuth2FeignRequestInterceptor implements RequestInterceptor {
+public class OAuth2FeignRequestInterceptor implements RequestInterceptor
+{
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(OAuth2FeignRequestInterceptor.class);
-    private static final String AUTH_HEADER_FORMAT = "%s %s";
+    private static final Logger LOGGER                       =
+                LoggerFactory.getLogger(OAuth2FeignRequestInterceptor.class);
+    private static final String AUTH_HEADER_FORMAT           = "%s %s";
```

Comment:
check formatting

Reply:
Updated files containing formatting issue as mentioned

Reply:
Same here as well. Kindly follow the code formatting guidelines.